extensions/zen-browser/package.json

```diff
@@ -5,11 +5,7 @@
   "description": "Search and open tabs from bookmarks and history in Zen Browser.",
   "icon": "zen-logo.png",
   "author": "Keyruu",
-  "contributors": [
```

Comment:
Why did you remove all contributors?

Reply:
Because they get pinged for every issue and they didn't contributed to this extension only for the Firefox extension. I initially included them because I copied a bunch of code from that extension.